SPECS/perl-File-pushd/perl-File-pushd.spec

```diff
@@ -0,0 +1,206 @@
+Name:           perl-File-pushd
+Version:        1.016
+Release:        8%{?dist}
+Summary:        Change directory temporarily for a limited scope
+License:        ASL 2.0
+Group:          Development/Libraries
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/File-pushd
+Source0:        http://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/File-pushd-%{version}.tar.gz
+BuildArch:      noarch
+# Module Build
+BuildRequires:  coreutils
+BuildRequires:  make
+BuildRequires:  perl
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 6.76
+# Module Runtime
+BuildRequires:  perl(Carp)
+BuildRequires:  perl(Cwd)
+BuildRequires:  perl(Exporter)
+BuildRequires:  perl(File::Path)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(File::Temp)
+BuildRequires:  perl(overload)
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+# Test Suite
```

Comment:
If these are test dependencies, can we gate their inclusion at build-time behind the %with_check variable we set during check builds?

Reply:
Added %with_check for tests